packages/cashier/src/Components/Form/payment-agent-deposit.jsx

```diff
@@ -20,7 +20,6 @@ const PaymentAgentDeposit = ({ onChangePaymentMethod, payment_agent_list, select
             <Text as='p' size='xs' weight='bold' color='prominent' className='payment-agent-list__list-header'>
                 <Localize i18n_default_text='Payment agents' />
             </Text>
-            <div className='payment-agent-list__list-line' />
```

Comment:
Which is better? Wrapping `<Text>` and `<div>` inside a container and giving it flex properties or `:after` pseudo element?

Reply:
Using `div` for border, as it used before, needs to define the width and margin of the line with percent, but using `:after` pseudo element it can be based on the width of the `Text`, so there is no need to control the width and margin of the line manually.

Reply:
We can remove the `div` as we are not using it anymore.